prometheus.yml

```
@@ -0,0 +1,8 @@
+global:
+  scrape_interval: 5s
+
+scrape_configs:
+  - job_name: 'competition-service'
+    metrics_path: '/actuator/prometheus'
+    static_configs:
+      - targets: ['host.docker.internal:19006']
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)